qiskit/providers/models/backendconfiguration.py

```diff
@@ -300,3 +340,115 @@ def __init__(self, backend_name, backend_version, n_qubits, basis_gates,
                          meas_lo_range=meas_lo_range, dt=dt, dtm=dtm,
                          rep_times=rep_times, meas_kernels=meas_kernels,
                          discriminators=discriminators, **kwargs)
+
+    @property
+    def dt(self) -> float:  # pylint: disable=invalid-name
+        """Time delta between samples on the signal channels in seconds."""
+        warnings.warn("The time delta is now returned in units of [s] rather than [ns].")
+        return self._dt * 1.e-9
+
+    @property
+    def dtm(self) -> float:
+        """Time delta between samples on the acquisition channels in seconds."""
+        warnings.warn("The time delta is now returned in units of [s] rather than [ns].")
+        return self._dtm * 1e-9
+
+    @property
+    def sample_rate(self) -> float:
+        """Sample rate of the signal channels in Hz (1/dt)."""
+        return 1.0 / self.dt
+
+    def hamiltonian(self) -> str:
+        """
+        Return the LaTeX Hamiltonian string for this device if provided.
+        """
+        return getattr(self, '_hamiltonian', {}).get('h_latex', None)
+
+    def hamiltonian_description(self) -> str:
+        """
+        Return the Hamiltonian description for this device if provided.
+        """
+        return getattr(self, '_hamiltonian', {}).get('description', None)
+
+    def drive(self, qubit: int) -> DriveChannel:
```

Comment:
For ease of use should this then also be exposed as a method directly on the backend, @ajavadia @dcmckayibm? This seems to make more sense to me.